package.json

```diff
@@ -41,7 +41,7 @@
     "test:e2e": "node tests/e2e/testRunner.js --development"
   },
   "dependencies": {
-    "@expensify/react-native-web": "0.18.10",
+    "@expensify/react-native-web": "0.18.11",
```

Comment:
We recently published v0.18.12 of our fork, can you update so we're testing on the latest package? 🙏